.gitignore

```diff
@@ -360,4 +360,5 @@ MigrationBackup/
 .ionide/
 
 # Fody - auto-generated XML schema
-FodyWeavers.xsd
\ No newline at end of file
+FodyWeavers.xsd
+/lexicon-garage3.Web/Views/Home/HomeIndex.cshtml
```

Comment:
Did you mean to add this to gitignore or was it a misclick?
I don't understand why it should be there.